Added Accrualify section (#574)
mattermost/mattermost-handbook: operations/finance/spending-company-money/how-to-use-accrualify/README.md

## What is Accrualify?

> As of December 1, 2020, the Finance team have implemented a new Procure-to-Pay automation software called Accrualify for the Procurement and Accounts Payable function. With this, you teams will have visibility into invoice status, approval workflows will be automated, and budgeting will be more streamlined. The purpose of this section is to provide step-by-step instructions on how to use Accrualify.

Reviewer: ```suggestion
As of December 1, 2020, the Finance team has implemented a new Procure-to-Pay automation system called Accrualify for Procurement and Accounts Payable workflows. With Accrualify, your teams will have visibility into invoice status, approval workflows will be automated, and budgeting will be more streamlined. The purpose of this section is to provide step-by-step instructions on how to use Accrualify.
```
